### PR TITLE
delete withdraw pattern in ReqEthereum

### PR DESCRIPTION
**Pull request by vrolland**

### CODE REVIEW

File: packages/requestNetworkSmartContracts/contracts/synchrone/RequestEthereum.sol [hunk 1]
```diff
@@ -557,9 +539,9 @@ contract RequestEthereum is RequestEthereumCollect {
 
 	/*
 	 * @dev Function internal to manage fund mouvement
-	 * @dev We had to chose between a withdraw pattern, a send pattern or a send + withdraw pattern and chose the last. 
-	 * @dev The withdraw pattern would have been a too big inconvenient for the UX. The send pattern would have allowed someone to lock a request. 
-	 * @dev The send + withdraw pattern will have to be clearly explained to users. If the payee is a contract which can let a transfer fail, it will need to be able to call a withdraw function from Request. 
+	 * @dev We had to chose between a withdraw pattern, a transfer pattern or a transfer+withdraw pattern and chose the transfer pattern.
```
Comment (romaric-juniet) [inline packages/requestNetworkSmartContracts/contracts/synchrone/RequestEthereum.sol]: Suggestion:
We had to chose between a withdrawal pattern, a transfer pattern or a transfer+withdrawal pattern and chose the transfer pattern. 
The withdrawal pattern would make UX difficult. The transfer+withdrawal pattern would make contracts interacting with the request protocol complex.
N.B.: The transfer pattern will have to be clearly explained to users. It enables a payee to create unpayable requests.

File: packages/requestNetworkSmartContracts/contracts/synchrone/RequestEthereum.sol [hunk 2]
```diff
@@ -557,9 +539,9 @@ contract RequestEthereum is RequestEthereumCollect {
...
 	/*
 	 * @dev Function internal to manage fund mouvement
-	 * @dev We had to chose between a withdraw pattern, a send pattern or a send + withdraw pattern and chose the last. 
-	 * @dev The withdraw pattern would have been a too big inconvenient for the UX. The send pattern would have allowed someone to lock a request. 
-	 * @dev The send + withdraw pattern will have to be clearly explained to users. If the payee is a contract which can let a transfer fail, it will need to be able to call a withdraw function from Request. 
+	 * @dev We had to chose between a withdraw pattern, a transfer pattern or a transfer+withdraw pattern and chose the transfer pattern. 
+	 * @dev The withdraw pattern would have been a too big inconvenient for the UX. The transfer+withdraw pattern would have complexify to much the other contracts around the request protocol.
+	 * @dev N.B: The send pattern will have to be clearly explained to users. The transfer pattern allow a payee to make a request unpayable.
 	 *
 	 * @param _requestId id of the request
```
Comment (romaric-juniet) [inline packages/requestNetworkSmartContracts/contracts/synchrone/RequestEthereum.sol]: requestId isn't used anymore

Comment (etiennetatur) [reply packages/requestNetworkSmartContracts/contracts/synchrone/RequestEthereum.sol]: complexify to much ==>too

File: packages/requestNetworkSmartContracts/contracts/synchrone/RequestEthereum.sol (hunk 1 above)
Comment (vrolland) [reply packages/requestNetworkSmartContracts/contracts/synchrone/RequestEthereum.sol]: updated

File: packages/requestNetwork.js/src/servicesContracts/requestEthereum-service.ts [hunk 3]
```diff
@@ -1167,37 +1167,7 @@ export default class RequestEthereumService {
         _fromBlock ?: number,
         _toBlock ?: number): Promise < any > {
         return new Promise(async (resolve, reject) => {
```
[the remark this answers is not in the document]
Comment (vrolland) [reply packages/requestNetwork.js/src/servicesContracts/requestEthereum-service.ts]: updated

File: packages/requestNetworkSmartContracts/contracts/synchrone/RequestEthereum.sol (hunk 2 above)
Comment (vrolland) [reply packages/requestNetworkSmartContracts/contracts/synchrone/RequestEthereum.sol]: updated

**Pull request merged**